sass/variables/_sizing.scss

```diff
@@ -73,6 +73,12 @@ $carousel-dot-spacing: 10px;
 $cta-icon-size: 2rem;
 $accordion-icon-size: 30px;
 $resource-continue-block-size: 40px;
+$visual-divider-endpoint-size: 14px;
+$visual-guide-step-size: 44px;
```

Comment:
in redlines this is 42px

![image](https://cloud.githubusercontent.com/assets/2359538/26641499/6e08f48e-45f0-11e7-9222-451e8574b362.png)